packages/shared/styles/components/organisms/SfStoreLocator.scss

```diff
@@ -58,69 +58,83 @@
     &__first-divider {
       --divider-display: block;
       --divider-max-width: unset;
-      --divider-margin: 8px 0;
-      @include for-desktop {
-        --divider-width: calc(100% + 2.5rem);
-      }
+      --divider-margin: 0 0 14px 0;
+    }
+    &__second-divider {
+      --divider-display: block;
+      --divider-max-width: unset;
+      --divider-margin: 14px 0 0 0;
     }
     &__item {
+      width: 100%;
+      padding: var(--store-item-padding, var(--spacer-xs) 0 var(--spacer-sm) 0);
       display: flex;
       flex-direction: var(--store-item-flex-direction, row);
       flex-wrap: var(--store-item-flex-wrap, nowrap);
       justify-content: var(--store-item-justify-content, flex-start);
       align-items: var(--store-item-align-items, flex-start);
```

Comment:
wdyt to remove these variables? are we need them ?

Reply:
removed :)